sound/soc/sof/mediatek/Kconfig

```diff
@@ -3,7 +3,7 @@
 config SND_SOC_SOF_MTK_TOPLEVEL
 	bool "SOF support for MTK audio DSPs"
 	depends on ARM64 || COMPILE_TEST
-	depends on SND_SOC_SOF_OF
+	select SND_SOC_SOF_OF
```

Comment:
I think it was intentional to 'hide' platforms like this.
If SOF_OF is not enabled then no platform is visible which is OF based, same for PCI and ACPI.

Reply:
Yes, but I could not find any simple way around it. Also the version here is wrong. I forgot to add my most recent changes to the last commit. The select should be in SND_SOC_SOF_(MTK|IMX)_COMMON. 

However, I think a similar behavior can be accomplished by adding SND_SOC_SOF_OF_TOPLEVEL etc. the same way the other similar problems have been solved, if that is considered important.